docs/NMOS With H.264.md

```diff
@@ -138,32 +181,230 @@ The SDP file at the `manifest_href` MUST comply with the requirements of RFC 618
 
 Therefore:
-- The `profile-level-id` format-specific parameters MUST be included with the correct value unless it corresponds to the default value. 
+- The `profile-level-id` format-specific parameters MUST be included with the correct value unless it corresponds to the default value. "Baseline" is the default profile value and "1" is the default level value.
 
 - The `packetization-mode` format-specific parameters MUST be included with the correct value unless it corresponds to the default value.
 
 - The `sprop-interleaving-depth`, `sprop-deint-buf-req`, `sprop-init-buf-time` and `sprop-max-don-diff` format-specific parameters SHOULD be included with the correct value if the `packetization-mode` equals one of the interleaved modes unless they correspond to their default value.
 
-- The `sprop-parameter-sets` MUST always be included if the Sender `parameter_sets_transport_mode` property is `out_of_band` and SHOULD be included if the property is `in_and_out_of_band`.
+- The `sprop-parameter-sets` MUST always be included if the Sender `parameter_sets_transport_mode` attribute is `out_of_band` and SHOULD be included if the attribute is `in_and_out_of_band`.
 
 The stream's active parameter sets MUST be compliant with the format-specific parameters, except `sprop-parameter-sets`, declared in the "fmtp=" attribute of an SDP transport file.
 
 If the Sender meets the traffic shaping and delivery timing requirements specified for ST 2110-22, the SDP transport file MUST also comply with the provisions of ST 2110-22.
 
+Informative note: ST 2110-22 does not require the `sampling` or `depth` SDP parameters. RFC 6184 does not define any such SDP parameters. The `sampling` and `depth` of the associated uncompressed picture could be derived from the H.264 active parameter sets by a Receiver.
+
 If the Sender meets the traffic shaping and delivery timing requirements specified for IPMX, the SDP transport file MUST also comply with the provisions of IPMX.
 
+An example SDP file is provided in the [Examples](../examples/).
+
 #### Other transports
-#### Multiplexed Flows
+
+For Nodes transmitting H.264 using other transports, the Sender resource MUST indicate the associated `urn:x-nmos:transport:` label of the transport or one of its subclassifications for the `transport` attribute.
+
+Sender resources provide no indication of media type or format, since this is described by the associated Flow resource.
+
+The `manifest_href` attribute MAY be `null` if an SDP transport file is not supported by the transport. Otherwise the SDP transport file MUST comply the transport specific requirements. There is no SDP format-specific parameters requirements for transports other than RTP.
```

Comment:
```suggestion
The `manifest_href` attribute MAY be `null` if an SDP transport file is not supported by the transport. Otherwise the SDP transport file MUST comply with the transport specific requirements. There is no SDP format-specific parameters requirements for transports other than RTP.
```

Reply:
done